micro_http/src/request.rs

```diff
@@ -104,7 +107,8 @@ impl<'a> RequestLine<'a> {
         (b"", b"", b"")
     }
 
-    fn try_from(request_line: &'a [u8]) -> Result<Self, RequestError> {
+    /// Tries to return a Request Line.
```

Comment:
Nope. Tries to parse a byte stream in a request line. Fails if the request line is malformed.